src/psyclone/core/variables_access_info.py

```diff
@@ -161,11 +171,11 @@ def options(self, key=None):
 
```

Comment:
Not the fault of this PR but I've just noticed the doc string (L160) says that this method returns None if the key is not specified in the options. That doesn't look right to me?

Reply:
I've updated the docs, I think this is correct, but not properly explained (it returns none if a valid key is queried that is not specified).

Reply:
Thanks.